ideascube/library/views.py

```diff
@@ -184,3 +184,34 @@ def get_row(self, book):
         return row
 
 book_export = staff_member_required(BookExport.as_view())
+
+
+class BookSpecimenExport(ZippedCSVExportMixin, View):
+
+    fields = ['item', 'barcode', 'serial', 'comments', 'location', 'file']
+    model = BookSpecimen
+    prefix = 'book_specimens'
+
+    def get_headers(self):
+        return self.fields
+
+    def get_row(self, bookspecimen):
+        row = {}
+
+        for field in self.fields:
+            if field == 'item':
+                row[field] = bookspecimen.item.name
```

Comment:
> We already know different books share one ISBN, so this can happen.

Not with our model:

```python
    # We allow ISBN to be null, but when it is set it needs to be unique.
    isbn = models.CharField(verbose_name=_('isbn'), max_length=40,
                            unique=True, null=True, blank=True)
```

Should I fix that too?